src/_includes/mini-toc.md

```diff
@@ -0,0 +1,13 @@
+{% if entries -%}
+Topics:
```

Comment:
@Sfshaza - I wasn't too sure what to use as a general heading. Since these pages won't be visited very often, maybe "Topics" is just fine. WDYT?

Reply:
Not sure what this is referring to, with the little snippet I see.  Where is mini_toc being used?